src/clef.ts

```diff
@@ -78,7 +78,7 @@ export class Clef extends StaveModifier {
         point: 0,
       },
       percussion: {
-        code: 'restMaxima',
+        code: 'unpitchedPercussionClef1',
```

Comment:
Is this a bug fix?

Reply:
Yes, the percussion clef was drawn using a rest maxima. This is what I meant with the first bulltet:

> Looking in detail at all the tests, I found two bugs:
> 
> * clef code used for percussion was wrong, now fixed.
> * stavetempo had not been aligned with [custom noteheads refactored #1477](https://github.com/0xfe/vexflow/pull/1477), now fixed

Reference
![Percussion Percussion_Clef Gonville](https://user-images.githubusercontent.com/22865285/205949058-42ec5546-21fd-4b1b-91c0-29d0f78a1214.png)
Current
![Percussion Percussion_Clef Gonville](https://user-images.githubusercontent.com/22865285/205948875-3fabcee6-a4c8-4525-985d-2e9f47f799f9.png)